Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+function add_vat(price){
+    const vatRate = 1.2;
+    const vatIncPri = price * vatRate
+    return `£${vatIncPri.toFixed(2)}`
```

Comment:
If we wanted to use the return value in calculation (e.g., `total = add_vat(pricd1) + price2`) instead of for output purpose, how would you change the function?